config/schema.yaml

```diff
@@ -311,10 +311,11 @@ SEARCH_CONFIGURATION:
   default:
     deploy: []
   examples:
-    - stage:
+    - comment: If `_merge` is true configuration merging with default
```

Comment:
`configuration merging with default` -> `configuration is being merged with the default`

Reply:
Thanks, changed.